app/src/main/java/mobdev/agrikita/controllers/ProductService.java

```diff
@@ -31,6 +45,25 @@ public ProductService(Context context) {
         serviceProductsApi = RetrofitClient.getClient(context).create(ProductServiceApi.class);
     }
 
+    public void uploadImage(MultipartBody.Part imagePart, UploadCallback callback) {
```

Comment:
Meant this
```suggestion
    public void uploadImage(MultipartBody.Part imagePart, UploadCallback callback) {
```

just take in a File